tests/unit/type/hash-field-expire.tcl

```diff
@@ -1200,8 +1200,8 @@ start_server {tags {"external:skip needs:debug"}} {
         test "HSETEX - Test 'EXAT' flag ($type)" {
             r del myhash
             r hset myhash f1 v1 f2 v2
-            assert_equal [r hsetex myhash EXAT 4000000000 FIELDS 1 f3 v3] 1
-            assert_range [expr [r httl myhash FIELDS 1 f3] + [clock seconds]] 3900000000 4000000000
```

Comment:
Do you mean that it may be at 0.5 seconds when httl is executed, but at 1.x seconds when [clock seconds] is executed, causing the test to fail?

Reply:
httl will return something equivalent to  `TTL - [clock seconds]`
then, we add `[clock seconds]` to it. Though, second `[clock seconds]` is one more than first one.